server/middleware/index.js

```diff
@@ -0,0 +1,24 @@
+import jwt from 'jsonwebtoken';
+
+const authenticate = {
+  Verify: (req, res, next) => {
```

Comment:
consistency please! You were told to work with **camel-casing** so why a pascal-case here